src/core/cgroup.c

```diff
@@ -2332,10 +2324,15 @@ static void unit_add_siblings_to_cgroup_realize_queue(Unit *u) {
                 Unit *m;
                 void *v;
 
+                slice->cgroup_members_mask_valid = false;
```

Comment:
this line deserves a comment

Reply:
or instead the big comment above should explicitly say what your comit msg say: this also implicitly invalidates all members masks

Reply:
This made me review my original removal of mask invalidation from `unit_load`. I realized that wouldn't be correct. I reorganized the commits a bit and added the explanation in the message of "cgroup: Implicit unit_invalidate_cgroup_members_masks".